docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -56,17 +58,32 @@ abstract class Client {
     encryptedMessage: string,
   }): Promise<NotifyMessage>;
   
+  // Generates and stores an identity key, indexed via the following key
+  // account-domain-isLimited
+  // Generates and returns SIWE message for said identity key
+  public abstract getSiweMessage(params: {
```

Comment:
Maybe `createSiweMessage` or `prepareSiweMessage` would be a better name?

Also would there be a desire to extract the identity key creation/storage into a separate function in-case they want to generate the SIWE statement themselves? @Elyniss you had a convo with Trust about this. IMO doesn't seem useful since the SIWE message is pretty fixed anyway.

Reply:
`format` doesn't feel like enough "action" to me. It generates an identity key and saves it which is somewhat important. It's like an impure preparation step, not just a pure formatting function.

Reply:
What do you think about `prepareRegistration`? 

Anything ending `SiweMessage` is still deceiving by that logic.

Reply:
> Also would there be a desire to extract the identity key creation/storage into a separate function in-case they want to generate the SIWE statement themselves? @Elyniss you had a convo with Trust about this. IMO doesn't seem useful since the SIWE message is pretty fixed anyway.

it's not fixed as statement is changed.

I personally prefer `formatSiweMessage`

Reply:
but the identity key registration is what makes me want to lean towards `prepareRegistration`. I'm fine with the latest changes